internal/zinc-core/src/main/scala/sbt/internal/inc/IncrementalCommon.scala

```diff
@@ -123,10 +123,6 @@ private[inc] abstract class IncrementalCommon(
       val nextInvalidations = result.nextInvalidations
       val current = result.analysis
 
-      val nextChangedSources: Set[VirtualFileRef] =
-        if (continue) javaSources
-        else Set.empty
```

Comment:
If we no longer have to do this, that would be great. This was a hack that was added to make Java symbols available to downstream Scala projects during pipelining. I wonder if this has been fixed via JAR merging implemented in https://github.com/sbt/zinc/pull/878?